### ISSUE TITLE
Is perceptual loss using a random initialized VGG? 

### ISSUE THREAD

**Issue by bladewaltz1** [NastyaMittseva/MoVQGAN#6]
Hi, thanks for sharing the code. [Here](https://github.com/ai-forever/MoVQGAN/blob/main/movqgan/modules/losses/lpips.py#L17) seems that you are using a `pretrained=False` VGG network. I wonder if this is the common practice? Thanks

**Comment by yinaoxiong** [comment]
> Hi, thanks for sharing the code. [Here](https://github.com/ai-forever/MoVQGAN/blob/main/movqgan/modules/losses/lpips.py#L17) seems that you are using a `pretrained=False` VGG network. I wonder if this is the common practice? Thanks

In fact, there is no need to load the pre-trained vgg, because the pre-trained vgg_lpips will be loaded later, which will overwrite the previous weights.

**Comment by bladewaltz1** [comment]
> > Hi, thanks for sharing the code. [Here](https://github.com/ai-forever/MoVQGAN/blob/main/movqgan/modules/losses/lpips.py#L17) seems that you are using a `pretrained=False` VGG network. I wonder if this is the common practice? Thanks
> 
> In fact, there is no need to load the pre-trained vgg, because the pre-trained vgg_lpips will be loaded later, which will overwrite the previous weights.

Thanks for your reply. Would you mind sharing me the code of loading the pre-trained vgg_lpips in this codebase?

**Comment by yinaoxiong** [comment]
> > > Hi, thanks for sharing the code. [Here](https://github.com/ai-forever/MoVQGAN/blob/main/movqgan/modules/losses/lpips.py#L17) seems that you are using a `pretrained=False` VGG network. I wonder if this is the common practice? Thanks
> > 
> > 
> > In fact, there is no need to load the pre-trained vgg, because the pre-trained vgg_lpips will be loaded later, which will overwrite the previous weights.
> 
> Thanks for your reply. Would you mind sharing me the code of loading the pre-trained vgg_lpips in this codebase?

pre-trained vgg_lpips will be auto download in this function https://github.com/ai-forever/MoVQGAN/blob/f3976d8752826c6ba003cfee8790e91ea23c57e8/movqgan/util.py#L48
